graphene_mongo/fields.py

```diff
@@ -110,8 +110,15 @@ def filter_args(self):
         if self._type._meta.filter_fields:
             for field, filter_collection in self._type._meta.filter_fields.items():
                 for each in filter_collection:
+                    filter_type = getattr(
+                        graphene, str(self._type._meta.fields[field].type).replace("!", ""))
+
+                    # Handle special where argument should be a list
+                    if each == 'in':
```

Comment:
Another way to implement it is to define a dict instead of extending `if-else` for more special case, such as:

```python
advanced_filter = {
    'in': lambda filter_type: graphene.List(filter_type)
}
if each in filter_type:
    filter_type = advanced_filter[each](filter_type)
```

Reply:
I like that, but is the `lambda` necessary?
```py
filter_type = getattr(...)

advanced_filter = {
    'in': graphene.List(filter_type)
}

if each in advanced_filter:
    filter_type = advanced_filter[each]
```

Reply:
Not really, your call :)

Reply:
if this way, we can even skip if statement:
```python
filter_type = advanced_filter.get(each, filter_type)
```

Reply:
Great. Sorry I haven't been able to come back to this, I'll try to fix it today.

Reply:
I did as you suggested and added the filters `'nin'` and `'all'` which accept lists as well.

I'm thinking later on I'll add a couple more keys to the advanced filters to cover all available query operators (there are a few boolean ones, for example), but on another PR with test cases.